src/bijectors/corr.jl

```diff
@@ -375,15 +374,18 @@ function _inv_link_chol_lkj(y::AbstractVector)
     T = float(eltype(W))
     logJ = zero(T)
 
+    z_vec = tanh.(y)
+    lc_vec = LogExpFunctions.logcosh.(y)
```

Comment:
This doesn't save any floating point operations, does it? Every element seems to be accessed exactly once in the nested loops below. Is it faster due to SIMD optimizations, or is there another reason to move it outside of the loop?

In general, I think you'd want to avoid broadcasting here. The operation is a simple `map` (so not actually broadcasting dimensions etc.) which tends to cause less stress on the compiler than `broadcast` and often leads to more optimized code.
```suggestion
    z_vec = map(tanh, y)
    lc_vec = map(LogExpFunctions.logcosh, y)
```

Reply:
It's the same number of operations, but a bit faster probably due to lower level optimisations (it wasn't immediately obvious to me why). Will change to `map`!